src/components/callout/callout.tsx

```diff
@@ -29,15 +31,11 @@ const CallOutText = styled.div`
 `;
 const CallOutIcon = styled.div`
   float: left;
-  width: 48px;
-  height: 48px;
   padding-right: 0.5em;
 
   ${up('sm')} {
     padding-right: 0;
     padding-left: 0.5em;
-    width: 72px;
-    height: 72px;
     flex: 0 0 72px;
```

Comment:
i remove the explicit sizing to rely on the illustration being sized appropriately. I removed the smaller variant,
as we don't have such a size for illustrations. It still looked good with that size. I would like to take that easy route as the 48px was a blind guess anyway before.


![Screen Shot 2021-11-16 at 15 00 12](https://user-images.githubusercontent.com/1701755/141999084-5ebe91cc-91b9-4b46-8b15-4baf7c2f1835.png)